client/blocks/eligibility-warnings/style.scss

```diff
@@ -5,6 +5,23 @@
     line-height: 1.5;
 }
 
+.is-placeholder {
```

Comment:
These style selectors need to be more specific to your use-case. It's causing weird ghosting in the media modal because we use the same modifier name:

![image](https://cloud.githubusercontent.com/assets/1779930/21437251/cbe0690c-c850-11e6-9e26-4cf44c3fa040.png)